stl/inc/xiosbase

```diff
@@ -547,9 +547,9 @@ private:
     fmtflags _Fmtfl; // format flags
     streamsize _Prec; // field precision
     streamsize _Wide; // field width
-    _Iosarray* _Arr; // pointer to first node of long/pointer array
-    _Fnarray* _Calls; // pointer to first node of call list
-    locale* _Ploc; // pointer to locale
+    _Iosarray* _Arr{nullptr}; // pointer to first node of long/pointer array
+    _Fnarray* _Calls{nullptr}; // pointer to first node of call list

```

Comment:
these can be not initialized in the constructors anymore after this addition.

Reply:
The constructor `ios_base()` doesn't initialize anything.

The member function `_Init()` assigns data members, but I don't think that it should be changed. It's fine if we redundantly assign data members, but I'm worried about unintended consequences of changing `_Init()` (I recall that it's "weird" but I don't know the details).